src/Exception/SerializableException.php

```diff
@@ -108,7 +108,7 @@ public function getPrevious()
      * @param  integer $line
      * @return array
      */
-    protected function filterTrace($trace, $file, $line)
+    protected function filterTrace(array $trace, string $file, int $line): array
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!